### PR TITLE
A few updates to catch up style guide.

### PR DESCRIPTION
**Pull request by evoskuil**

### CODE REVIEW

File: style_guide.md [hunk 1]
```diff
@@ -108,6 +106,16 @@ Don't tolerate warnings and don't suppress them due to laziness.
 
 Use the following source file layouts.
 
+Precede comments with a blank line.
```
Comment (RojavaCrypto) [inline style_guide.md]: This isn't allowed?

```cpp
// Something something
something()
// Then this happens.
continued()
```

Comment (RojavaCrypto) [inline style_guide.md]: #407

Comment (evoskuil) [inline style_guide.md]: Right, this has been applied fairly consistently, though possibly not in some sections that haven't been hit in a long time. In order to open up the code vertically, and to do so consistently, I applied two commonly applied rules: scopes and comments are preceded by blank lines.

Comment (evoskuil) [inline style_guide.md]: This is more accurately: the closing of a scope is followed by a blank line, I'll update.

File: style_guide.md [hunk 2]
```diff
@@ -104,18 +104,18 @@ Don't use global variables or `goto`, we are grown-ups now.
...
-
 Use //// to temporarily disable code.
 
 Use /// for public API documentation (short form) or javadoc format.
 
 Use // for source code commentary.
 
+Follow the closing of a scope with a blank line before a subsequent statement.
+
+Precede comments with a blank line.
```
Comment (RojavaCrypto) [inline style_guide.md]: All lines with a comment should have a blank line before it? Sounds ambiguous.
#407

**Pull request merged**